app/models/gi_bill_feedback.rb

```diff
@@ -20,6 +34,50 @@ def get_user_details
     { 'profile_data' => profile_data }
   end
 
+  def remove_malformed_options(options_hash)
```

Comment:
Is there a particular reason that `#remove_malformed_options` and `#transform_malformed_options` need to be public methods here?

Reply:
no, i'll make them private